src/Jackett.Common/Definitions/torrentmasters.yml

```diff
@@ -0,0 +1,151 @@
+    - {id: 73, cat: Console, desc: "Játék - Konzol"}
+    - {id: 94, cat: PC, desc: "Játék - RIP"}
+    - {id: 79, cat: PC/Mobile-Other, desc: "Mobil/PDA"}
+    - {id: 95, cat: PC, desc: "Programok - Egyéb (Win)"}
+    - {id: 74, cat: PC/ISO, desc: "Programok - ISO (Win)"}
+    - {id: 75, cat: PC/Mac, desc: "Programok - Linux/Unix/Mac OS"}
+    - {id: 10, cat: TV, desc: "Sorozat - DVD (ENG)"}
+    - {id: 10, cat: TV, desc: "Sorozat - DVD (HUN)"}
+    - {id: 89, cat: TV/HD, desc: "Sorozat - HD (ENG)"}
+    - {id: 97, cat: TV/HD, desc: "Sorozat - HD (HUN)"}
+    - {id: 86, cat: TV, desc: "Sorozat - XviD (ENG)"}
+    - {id: 90, cat: TV, desc: "Sorozat - XviD (HUN)"}
+    - {id: 98, cat: XXX, desc: "XXX - HD"}
+    - {id: 99, cat: XXX/DVD, desc: "XXX - DVD"}
+    - {id: 76, cat: XXX/ImageSet, desc: "XXX - Képek"}
+    - {id: 69, cat: XXX/XviD, desc: "XXX - XviD"}
+    - {id: 70, cat: Audio, desc: "Zene - HUN"}
+    - {id: 71, cat: Audio, desc: "Zene - Külföld"}
+
+  modes:
+    search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: 0
+    options:
+      0: added
+      1: name
+      3: comments
+      5: size
+      6: grabbed
+      7: seeders
+      8: leechers
+
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+
+login:
+  method: post
+  path: takelogin.php
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+
+  test:
+    selector: a[href^="logout.php"]
+
+search:
+  paths:
+    - path: letoltes.php
+  inputs:
+    # doesn't seem to support selecting multiple categories
+    # if multiple categories are selected only one of them will be used
+    $raw: "{{ range .Categories }}cat={{.}}&{{end}}"
+    search: "{{ .Keywords }}"
+    # 0 active, 1 active and dead, 2 deadonly,  3 free torrents
+    # 4 x2 torrents, 5 waiting for seed, 6 user torrents, 7 uploaded on request
+    incldead: 1
+    sort: "{{ .Config.sort }}"
+    type: "{{ .Config.type }}"
+    # does not support imdbid searches
+
+  rows:
+    selector: table > tbody#linkhatter > tr:nth-child(1)
+
+  fields:
+    category:
+      selector: a[href^="letoltes.php?cat="]
+      attribute: href
+      filters:
+        - name: querystring
+          args: cat
+    title:
+      selector: td:nth-child(3) > a[href^="torrent_adatok.php"]
+      attribute: title
+    details:
+      selector: a[href^="torrent_adatok.php?id="]
+      attribute: href
+    download:
+      selector: a[href^="download.php?id="]
+      attribute: href
+    description:
+      selector: td:nth-child(3) font[color="gray"]
+
+    date:
+      selector: td:nth-child(5) nobr
+      filters:
+        - name: replace
+          args: ["\xA0", ""]
+        - name: regexp
+          args: "(....-..-....:..:..)"
```

Comment:
It's probably cleaner to do a replace here like in punkshorror

Reply:
I don't think we can avoid using a regex. This is what the element looks like:
```html
<nobr>2021-11-05&nbsp;12:57:06 <br><font size="small">(1 órája)</font></nobr>
```

Reply:
```suggestion
          args: "$(.+?) "
```
Does that work?

Reply:
I'm not that good with regular expressions. What does the question mark mean in this case?

Reply:
https://duckduckgo.com/?q=regex+cheatsheet&ia=cheatsheet&iax=1
but I expect the leading `$` should be a `^` instead

an alternative to the regexp would be to use the remove and drop the font, something like
```yaml
    date:
      selector: td:nth-child(5) nobr
      remove: font
      filters:
        - name: replace
          args: ["\xA0", ""]
        - name: append
          args: " +01:00" # CET
        - name: dateparse
          args: "2006-01-0215:04:05 -07:00"
```

Reply:
I think that is a bit cleaner, committed.

Reply:
Huh, did not realise `remove` was even a thing.

Yep, that was supposed to have been a `^` but seemingly I wasn't paying attention ;P

Reply:
>Huh, did not realise `remove` was even a thing.

Hah! About time you RTFM don't you think ;-b
Its described [here](https://github.com/Jackett/Jackett/wiki/Definition-format#search-html) just after the first search example in the description selector, and with explanation text below it. ;-)

Reply:
lol I can't say I've read it start to finish at any point, just skimmed it and referred back to specific parts when needed.